pkg/objstore/gcs/gcs.go

```diff
@@ -124,7 +124,7 @@ func (b *Bucket) Exists(ctx context.Context, name string) (bool, error) {
 }
 
 // Upload writes the file specified in src to remote GCS location specified as target.
-func (b *Bucket) Upload(ctx context.Context, name string, r io.Reader) error {
+func (b *Bucket) Upload(ctx context.Context, name string, r io.Reader, sizeBytes int64) error {
```

Comment:
I am pretty sure we can reuse this knowledge, but you can leave that to others. Just please use _ for arg name that is not used.